logbook-api/src/main/java/org/zalando/logbook/attributes/RequestAttributesExtractor.java

```diff
@@ -0,0 +1,17 @@
+package org.zalando.logbook.attributes;
+
+import org.apiguardian.api.API;
+import org.zalando.logbook.HttpRequest;
+
+import javax.annotation.Nonnull;
+
+import static org.apiguardian.api.API.Status.STABLE;
+
+@API(status = STABLE)
+@FunctionalInterface
+public interface RequestAttributesExtractor {
```

Comment:
Are you planning to add this concept to the `logbook-spring-boot-autoconfigure` module? So that extractors can be registered as beans and configured via properties, e.g. claim names could be controlled via properties.

Reply:
Definitely, it's on my ToDo list for this PR.